src/directives/colorpicker.js

```diff
@@ -0,0 +1,103 @@
+goog.provide('ngeo.ColorpickerController');
+goog.provide('ngeo.colorpickerDirective');
+
+goog.require('ngeo');
+
+
+ngeo.module.value('ngeoColorpickerTemplateUrl',
+    /**
+     * @param {angular.JQLite} element Element.
+     * @param {angular.Attributes} attrs Attributes.
+     * @return {string} Template URL.
+     */
+    function(element, attrs) {
+      var templateUrl = attrs['ngeoColorpickerTemplateurl'];
+      return templateUrl !== undefined ? templateUrl :
+          ngeo.baseTemplateUrl + '/colorpicker.html';
+    });
+
+/**
+ * Provides the "ngeoColorpicker" directive, a widget for
+ * selecting a color among predefined ones.
+ *
+ * Example:
+ *
+ *     <div ngeo-colorpicker="ctrl.colors">
+ *     </div>
+ *
+ *
+ * @param {string|function(!angular.JQLite=, !angular.Attributes=)}
+ *     ngeoColorpickerTemplateUrl Template URL for the directive.
+ * @return {angular.Directive} Directive Definition Object.
+ * @ngInject
+ * @ngdoc directive
+ * @ngname ngeoColorpicker
+ */
+ngeo.colorpickerDirective = function(ngeoColorpickerTemplateUrl) {
+  return {
+    restrict: 'A',
+    scope: {
+      colors: '<?ngeoColorpicker',
+      color: '=?ngeoColorpickerColor'
+    },
+    controller: 'NgeoColorpickerController',
+    controllerAs: 'ctrl',
+    bindToController: true,
+    templateUrl: ngeoColorpickerTemplateUrl
+  };
+};
+
+
+ngeo.module.directive('ngeoColorpicker', ngeo.colorpickerDirective);
+
+/**
+ *
+ * @type {Array.<Array.<string>>}
+ * @const
+ */
+var defaultColors = [
+  ['#880015', '#ed1c24', '#ff7f27', '#fff200', '#22b14c', '#00a2e8',
+    '#3f48cc', '#a349a4'],
+  ['#b97a57', '#ffaec9', '#ffc90e', '#efe4b0', '#b5e61d', '#99d9ea',
+    '#7092be', '#c8bfe7'],
+  ['#ffffff', '#f7f7f7', '#c3c3c3', '#000000']
```

Comment:
I don't find where he is visible, In the redlining I have a colorpicker from the browser...

I see that it's overloaded in the example but I think that the default value should already be a good chose :-)